src/assets/scss/_common.scss

```diff
@@ -1,9 +1,16 @@
+* {
+  box-sizing: border-box;
+}
 body {
   font-family: verdana;
   font-size: 16px;
   background: #3f94ff;
 }
-
+h3 {
+  font-size: 1.17em;
+  margin: 1em 0px;
+  font-weight: bold;
+}
 .container {
   margin: 20px auto 20px auto;
```

Comment:
Either use 'EM' or 'PX'

Reply:
updated

Reply:
I don't see the changes

Reply:
I think code was not pushed properly kindly take a look now